### ISSUE TITLE
Try using less secure emails than Gmail for the College Shirts Project.

### ISSUE THREAD

**Issue by NotAqua** [hoppuman/TikTokHacks#5]
I am getting a issue where it says the login is incorrect, but it is and i've logged in multiple times. Gmail is blocking logins due to suspicious activity due to the method. 2FA is off, and nothing should be bothering. I even read over some support forum articles, and here's what i've seen and it doesn't work. I think there should be a fix by adding it as a request feature. I even went to the extend of making a brand new Gmail account and it still is having the same issue.

Forum: https://support.google.com/mail/answer/7126229?p=BadCredentials&visit_id=637301336350965380-3323792305&rd=2#cantsignin

Screenshot of Error: https://i.imgur.com/yDwy3OZ.png

Screenshot of Security issue: https://i.imgur.com/pUpXCTj.png

**Comment by CraXell** [comment]
To use gmail in  you need to have 2FA from what i know.
I used to make a similar project and on one site you can add custom passwords just for gmail only, You can try using Gmail's API.

**Comment by NotAqua** [comment]
> To use gmail in you need to have 2FA from what i know.
> I used to make a similar project and on one site you can add custom passwords just for gmail only, You can try using Gmail's API.

Nope, still caused same issues. I've read everywhere and almost everything on this, and nothing was fixed.

**Comment by CraXell** [comment]
Have no idea right now.

**Comment by ethanbreck** [comment]
I might try and create a gmail API thing, but that would mean people would need to mess with google's API and download their own API key and stuff.
